src/transformStateWithClones.js

```diff
@@ -6,8 +6,79 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+
+  let currentState = { ...state };
```

Comment:
[NAMING] - The variable name 'currentState' is appropriate as it represents a copy of the state. Good job on using a proper name for the object copy.